### PR TITLE
send job decision to cinder via their api

### LINKED ISSUE #21660: Implement cinder decision resolution from AMO
prerequisite of #21576 (that I thought initially would be trivial enough not to need to land separately).  We need the ability to create a decision on abuse reports that have been processed via amo reviewer tools, and notify cinder of that decision.

It turns out the at a decision on an entity does not automatically resolve a job - and a job can't be specified in the decision - so the job will be closed manually also.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/ADDSRV-641)

### PR DESCRIPTION
**Pull request by eviljeff**
fixes #21660 - needed for #21576

### CODE REVIEW

File: src/olympia/abuse/models.py [hunk 1]
```diff
@@ -44,6 +45,10 @@ class CinderJob(ModelBase):
         ('AMO_DELETE_COLLECTION', 6, 'Collection delete'),
         ('AMO_APPROVE', 7, 'Approved (no action)'),
     )
+    DECISION_ACTIONS.add_subset(
+        'UNRESOLVED',
+        ('NO_DECISION', 'AMO_ESCALATE_ADDON'),
```
Comment (diox) [inline src/olympia/abuse/models.py:50]: That looks weird. Is that what you meant ?

**Review by diox — APPROVED** [review]

File: src/olympia/abuse/models.py (hunk 1 above)
Comment (eviljeff) [reply src/olympia/abuse/models.py:50]: It was - does it not make sense? These are reports that need a decision still.

Comment (diox) [reply src/olympia/abuse/models.py:50]: You're right, misread that

File: src/olympia/abuse/cinder.py [hunk 2]
```diff
@@ -144,6 +144,42 @@ def appeal(self, *, decision_id, appeal_text, appealer):
...
+    def create_decision(self, *, review_text, policy_uuids):
+        if self.type is None:
+            # type needs to be defined by subclasses
+            raise NotImplementedError
+        url = f'{settings.CINDER_SERVER_URL}create_decision'
+        headers = {
+            'accept': 'application/json',
+            'content-type': 'application/json',
+            'authorization': f'Bearer {settings.CINDER_API_TOKEN}',
+        }
```
[the remark this answers is not in the document]
Comment (eviljeff) [reply src/olympia/abuse/cinder.py]: https://github.com/mozilla/addons-server/pull/21661/commits/9d058f219981e88e70e042e10e5c290f1d958aaf

**Pull request merged**